BEST_PRACTICES.md

```diff
@@ -127,7 +127,39 @@ Multiple-lines:
  */
 ```
 
-## Blocks, delegates or data source
+## Blocks, delegates and data source
+
+## Naming
+
+Declaring delegates for some UIViewControllers, for example UITableViewControllers or UICollectionViewControllers, can be annoying since they have their own delegates that use the variable `delegate` and can't be overwritten.
+
+Usually this leads to the developer naming the delegate things like `recipesDelegate`, `recipesControllerDelegate` and others. This causes to confusion for the API user, since now there are two delegates. We avoid this by not subclassing directly UITableViewController or UICollectionViewController, instead we use SweetTableController and SweetCollectionController, by doing this we can use the `delegate` variable.
+
+```swift
+protocol RecipesControllerDelegate: class {
+}
+
+class RecipesController: SweetTableController/SweetCollectionController {
+    var delegate: RecipesControllerDelegate
+}
+```
+
+If you are using something else than UITableViewController and UICollectionViewController, first consider making a simple wrapper and submitting a PR to the [SweetUIKit](https://github.com/UseSweet/SweetUIKit) library. If this is not possible use another meaningful name, such as the name of the protocol or the class.
```

Comment:
Missing comma: `if this is not possible, user another meaning name`. Also _another_ here is very out of place. Why would I use _another_ meaningful name if I already had one? You know what I mean? Should probably be`use a meaningful name`. 😄